bin/docker-build

```diff
@@ -22,4 +22,4 @@ fi
 "$bindir"/docker-build-grafana
 "$bindir"/docker-build-jaeger-webhook
 "$bindir"/docker-build-metrics-api
-
+"$bindir"/docker-build-tap
```

Comment:
nit: new line?

Reply:
As in add a newline? I think it's fine since there is no spacing between the other commands.

Reply:
Previously this file ended in two newline characters.  After your change it now ends in one newline character.  I think that's fine.